doc/manual/src/glossary.md

```diff
@@ -1,5 +1,15 @@
 # Glossary
 
+  - [installable]{#gloss-installable}\
+    Argument type that can be passed to [`nix-build`](./command-ref/nix-build.md) or [`nix build`](./command-ref/new-cli/nix3-build.md) (experimental):
+
+    - path to Nix expression
```

Comment:
Does this mean

- filesystem path of a `.nix` file (e.g. `~/nixpkgs/default.nix`) or directory containing `default.nix` or
- attribute path (e.g. `pkgsStatic.busybox`)

I guess both are correct, depending on which command-line flags (`-f`, `-A`) and cli (`nix-build`, `nix build`) you use.

Aside: it has always seemed weird to me that `nix` needs flags in order to refer to an attrpath within a `.nix` file instead of being able to write `nix build ~/nixpkgs/default.nix/pkgsStatic.busybox`.

Reply:
The specifics of how things may be weird are not a concern right here, I'm just trying to state the facts.

You see, writing this down can help determining what's weird and which ways there may be to change that. But let's first focus on on the part of writing down clearly and correctly.

Reply:
Agree completely, I *don't* want to see that fixes in this docs PR!

Reply:
Sorry, I should've resisted the urge to bikeshed there.  I've crossed out the distracting part of my comment.